### ISSUE TITLE
No longer able to drag a PDF file to an entry

### ISSUE THREAD

**Issue by anassal** [JabRef/jabref#234]
2.11b4 on Windows 8.1:

No longer able to drag a PDF file to an entry (e.g. from Windows Explorer) as before. A stop sign appears instead.

**Comment by koppor** [comment]
I tried to reproduce here on Windows 10. Drag'n'drop works perfectly. If I run JabRef as admin mode, I can't drag: A blue sign which is crossed with one bar appears.

Which Java version are you running?

Please open a command line (not as administrator!) and run `java -jar JabRef-2.11b4.jar` and try drag and drop again.

**Comment by lenhard** [comment]
Should we close this, since @anassal seems to have lost interest?

**Comment by ajbelle** [comment]
Just following this up, I had the same when I braved installing JabRef3.2 on Winn7 with Java8.73. I would show the pop-up asking to import meta-data but not go on to drag and drop the file in.  I uninstalled back to v3.0 and was going to report when found this issue153. I reinstalled to follow @koppor  suggestion about Java, but now it is working. No difference between my installing as far as I know, so perhaps the problem "went away" for anassal also.

**Comment by lenhard** [comment]
That's good to hear! Thanks a lot @ajbelle for following up on this!

**Comment by ajbelle** [comment]
@lenhard sorry to report back that is "came back". After trail and error it seems the the recent changes to "SEARCH" mean that you **cannot drag and drop a file to any record that is, or was, a search hit**! Completely the reverse to what you want.
I have reverted back to version 3.0 and it works fine (I haven't used Ver3.1).
I hope this helps you identify the issue and coding problem. My previous posting give install details.

**Comment by anassal** [comment]
Hi, I've been very busy to follow-up this issue, but I increasingly interested about the great JabRef!

I no long have this issue on 3.2 and Java "1.8.0_74", even when attaching a PDF to a search hit.

` Java(TM) SE Runtime Environment (build `1.8.0_74-b02)
` Java HotSpot(TM) 64-Bit Server VM (build 25.74-b02, mixed mode)

**Comment by lenhard** [comment]
@ajbelle oh, my bad! I guess we should try to reproduce this issue then.

**Comment by ajbelle** [comment]
I installed 3.3dev-snapshot-2016-03-25-livesearch-55fc5f2 provided by @oscargus to check issue  [https://github.com/JabRef/jabref/issues/1034](url) . The drag and drop problem appears to have been resolved. I reinstalled V3.0 and 3.2 several times but this did not allow me to drag and drop, as explained above. Anyhow my install on Win7 Enterprise is now working with the livesearch version. It was related to `Search`, so perhaps the glitch was in that code base.
